docs/how-tos/set-configuration.md

```diff
@@ -59,4 +59,4 @@ config = Config.from_file(config_path)
 
 !!! note
 
-    In the Python API, the `Rag.chat()` function also allows you to set certain RAG-specific configurations like `document` and `assistants`.
+    In the Python API, the [`Rag.chat()` function](../references/python-api.md#ragna.core.Chat) also allows you to set certain RAG-specific configurations like `document` and `assistants`.
```

Comment:
I think you can use they dynamic link here

```suggestion
    In the Python API, the [ragna.Rag.chat][] function also allows you to set certain RAG-specific configurations like `document` and `assistants`.
```

Reply:
Renders and links correctly:

https://ragna--197.org.readthedocs.build/en/197/how-tos/set-configuration/#set-configuration-using-the-file

![image](https://github.com/Quansight/ragna/assets/6849766/8be9277e-74e9-4934-a27a-210a868291cd)

Reply:
TIL I've never seen that feature in `mkdocs` 👀

Reply:
This is not a feature of vanilla `mkdocs`, but rather the [`mkdocstrings` plugin](https://mkdocstrings.github.io/usage/#cross-references)

https://github.com/Quansight/ragna/blob/4962665828c51bc6ef0f2d772574f08caec547a8/mkdocs.yml#L45-L56

Reply:
Thanks for the reference! I've primarily used `mkdocs` for Argo Workflows and hadn't gotten too into the plugin ecosystem as we're looking to changed to RTD for versioned docs https://github.com/argoproj/argo-workflows/issues/11390